src/Interfaces/Options/Particles/IOpacityAnimation.ts

```diff
@@ -0,0 +1,12 @@
+export interface IOpacityAnimation {
+    enable: boolean;
+    speed: number;
+
+    /**
+     * @deprecated use the new minimumValue instead
+     */
+    opacity_min: number;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'opacity_min' is not in camel case.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5139404)